kubernetes_deploy/production/dashboard.yml

```diff
@@ -447,6 +447,6 @@ data:
       },
       "timezone": "browser",
       "title": "LAA CLA / FALA",
-      "uid": "87a9fc9281061289879d05928d54a80c6b57818a",
+      "uid": "Zr8wbp3iz",
```

Comment:
Is this `uid` change related/needed?

Reply:
The previous uuid was being used(due to copy/paste).
The new Grafana export has different / newer uuid format.

As for whether the uuid is needed, I'm not sure. The Grafana export always has a uuid